hilti/toolchain/include/compiler/unit.h

```diff
@@ -385,6 +385,9 @@ class Unit {
     // Recursively destroys the module's AST.
     void _destroyModule();
 
+    // Helper for depencencies() to recurse.
+    void _recursiveDependencies(std::vector<std::weak_ptr<Unit>>* dst, std::vector<const Unit*>* seen);
+
```

Comment:
Can we instead move this to a helper function in the `.cc` file and pass in the `Unit` arg via a parameter `const Unit&`?

Reply:
The function needs access to the `_dependencies` member, so we'd need to friend it, which doesn't seem like it would help much.